shared/src/http.rs

```diff
@@ -161,3 +163,16 @@ mod tests {
         assert!(filtered.get("custom").is_none());
     }
 }
+
+pub fn make_error_response<E>(status_code: StatusCode) -> Response<BoxBody<Bytes, E>>
```

Comment:
this is just copy/paste - moved as-is from `proxy` module so this helper function can be used in the ingest-router as well